tools/wptserve/tests/functional/test_request.py

```diff
@@ -117,7 +118,7 @@ def handler(request, response):
     def test_non_ascii_in_headers(self):
         @wptserve.handlers.handler
         def handler(request, response):
-            return request.headers["foo"]
+            return request.headers[b"foo"]
```

Comment:
This is really a drive-by change to demonstrate the recommended way of accessing headers, even though the getter accepts both binary and text strings.